packages/quantic/force-app/examples/main/lwc/exampleRelevantGenerativeAnswering/exampleRelevantGenerativeAnswering.js

```diff
@@ -0,0 +1,82 @@
+import {LightningElement, api} from 'lwc';
+// @ts-ignore
+import caseTemplate from './resultTemplates/caseResultTemplate.html';
+// @ts-ignore
+import chatterTemplate from './resultTemplates/chatterResultTemplate.html';
+// @ts-ignore
+import childTemplate from './resultTemplates/childTemplate.html';
+// @ts-ignore
+import parentTemplate from './resultTemplates/parentTemplate.html';
+// @ts-ignore
+import youtubeTemplate from './resultTemplates/youtubeResultTemplate.html';
+
+export default class ExampleRelevantGenerativeAnswering extends LightningElement {
+  /** @type {string} */
+  @api engineId = 'example-relevant-generative-answering';
+  /** @type {string} */
+  @api searchHub = 'default';
+  /** @type {string} */
+  @api pipeline = 'genqatest';
```

Comment:
set the pipeline to `genqatest` so that it could generate answers. However, I was wondering if we should not add a note on the page to try to use : `How to resolve netflix connection with tivo` as a query? Otherwise someone could make queries that does not generate an answer, which defeats the purpose of this example...

Reply:
This component could serve as code sample that we can share with the clients, so it will not be necessarily deployed, people would just be curious to see how the `c-quantic-generated-answer` can be added to their HTML in order to appear in the search page.
But we could add a comment mentioning the query that could yield a generated answer with this pipeline.